XLS-0033d-multi-purpose-tokens/README.md

```diff
@@ -983,3 +1011,59 @@ As can be seen from the following size comparison table, Trustlines take up appr
 | Bytes for holding 10 Tokens |         1         |       1,450       |            2             |          3,260          |             2.2x              |
 | Bytes for holding 32 Tokens |         2         |       5,556       |            4             |         12,264          |             2.2x              |
 | Bytes for holding 64 Tokens |         3         |      13,070       |            6             |         28,444          |             2.2x              |
+
+### 2.3.2. `STAmount` serialization
+Referenced from https://gist.github.com/sappenin/2c923bb249d4e9dd153e2e5f32f96d92 with some modifications:
+
+#### Binary Encoding
+To support this idea, we first need a way to leverage the current [STAmount](https://xrpl.org/serialization.html#amount-fields) binary encoding. To accomplish this, we notice that for XRP amounts, the maximum amount of XRP (10^17 drops) only requires 57 bits. However, in the current `XRP` STAmount encoding, there are 62 bits available. So, so we can repurpose one of these bits to indicate if an amount is indeed a MPT or not (and still have 4 bits left over for future use, if needed). 
+
+This enables MPT amounts to be represented in the current `STAmount` binary encoding.  he rules for reading the binary amount fields would be backward compatible, as follows:
+
+1. Parse off the Field ID with a type_code (`STI_AMOUNT`). This indicates the following bytes are an `STAmount`.
+2. Inspect the next bit. If its value is `1`, then continue to the next step. If not, then this `STAmount` does **not** represent an MPT nor XRP (instead this is a regular IOU token amount, and can be parsed according to existing rules for those amounts). 
+3. Ignore (for now) the 2nd bit (this is the sign-bit, and is always 1 for both XRP and MPT).
+4. Inspect the 3rd bit. If `0`, then parse as an XRP value per usual. However, if `1`, then parse the remaining `STAmount` bytes as an MPT.
+
+#### Encoding for XRP Values (backward compatible)
+
+```
```

Comment:
After talking to @shawnxie999 we agreed to adjust this section to be this:

```
┌───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│ ┌──────────────────┐ ┌──────────────────────┐ ┌──────────────────┐ ┌──────────────────┐ ┌───────────────────────────────────────────┐ │
│ |        0         | |          1           | |        0         | |                  | |                                           | │
│ |                  | |                      | |                  | |     Reserved     | |                XRP Amount                 | │
│ |  "Not XRP" bit   | | Sign bit (always 1;  | |   "IsMPT" bit    | |     (4 bits)     | |                 (57 bits)                 | │
│ | 0=XRP/MPT; 1=IOU | |      positive)       | |   0=XRP; 1=MPT   | |                  | |                                           | │
│ └──────────────────┘ └──────────────────────┘ └──────────────────┘ └──────────────────┘ └───────────────────────────────────────────┘ │
└───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```

Once that change is made, I'll merge this PR.

Reply:
LGTM